src/module/actor/data-model-character.js

```diff
@@ -70,10 +70,7 @@ export default class OseDataModelCharacter extends foundry.abstract.DataModel {
       details: new ObjectField(),
       ac: new ObjectField(),
       aac: new ObjectField(),
-      encumbrance: new SchemaField({
-        value: new NumberField({ integer: false }),
-        max: new NumberField({ integer: false }),
-      }),
+      encumbrance: new ObjectField(),
```

Comment:
Would you mind trying something?
```suggestion
      encumbrance: new SchemaField({
        max: new NumberField({ initial: 1600 }),
      }),
```
See if that fixes it, as well. My guess is that we haven't been setting an initial encumbrance max all this time.

If it doesn't work, eh, heck with it, we'll keep it as you have it. If it does, then maybe we add a system setting for default initial value for max carry weight?

Reply:
Huh. Fair enough. It was worth a try.

We may want to add a migration to ensure that null/undefined values for `encumbrance.max` are set to `1600` when this goes out. That default gets pulled from `template.json` if we're not specifically setting a schema field up for it.